packages/@guardian/source-react-components-development-kitchen/components/summary/ErrorSummary.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { ErrorSummary, ErrorSummaryProps } from './ErrorSummary';
+import type { Story } from '../../../../../lib/@types/storybook-emotion-10-fixes';
+import {
+	asPlayground,
+	asChromaticStory,
+} from '../../../../../lib/story-intents';
+
+export default {
+	title: 'Kitchen/source-react-components-development-kitchen/Error Summary',
+	component: ErrorSummary,
+	args: {
+		error: 'There has been a problem',
```

Comment:
Minor thing, but if you include the `context` prop here with an empty string:

```suggestion
		error: 'There has been a problem',
		context: '',
```

you get text input field for this prop in the Docs page, rather than a button that reveals a text input field

**Before**

![Screenshot 2021-10-01 at 13 45 39](https://user-images.githubusercontent.com/5931528/135621894-b61937a6-f966-46cc-a2a5-df4629f8cb3e.png)

**After**

![Screenshot 2021-10-01 at 13 45 24](https://user-images.githubusercontent.com/5931528/135621889-56ed1fbd-31d3-42cf-9dde-b58908cc6359.png)